great_expectations/expectations/expectation.py

```diff
@@ -1699,20 +1700,32 @@ def _validate_metric_value_between(
         runtime_configuration: dict = None,
         execution_engine: ExecutionEngine = None,
     ):
-        metric_value = metrics.get(metric_name)
+        metric_value: Optional[Any] = metrics.get(metric_name)
```

Comment:
Would it be better to just leave this out?

Reply:
@Kilo59 I prefer to have a type hint of some kind, rather than none (to reduce the cognitive strain to the reader).